lib/logflare/application.ex

```diff
@@ -24,8 +24,8 @@ defmodule Logflare.Application do
     children = get_children(env)
 
     if Application.get_env(:logflare, :opentelemetry_enabled?) do
-      :opentelemetry_cowboy.setup()
-      OpentelemetryPhoenix.setup(adapter: :cowboy2)
+      OpentelemetryBandit.setup()
```

Comment:
You were worried about too much data from ingest traces.

This gets us otel for egress but does not ship traces for all ingest requests?

Reply:
I realised that I had excluded the ingest and endpoint route traces in the sampler 😂
I think I'll add back the routes but with a different sample ratio env var